cluster-provision/k8s/1.24/provision.sh

```diff
@@ -47,8 +47,26 @@ function pull_container_retry() {
     fi
 }
 
+# FIXME: remove after version >= kernel-4.18.0-492.el8
```

Comment:
I think this is the major change on this PR and should appear please on the PR desc

can we consider using koji ?

https://discussion.fedoraproject.org/t/downgrading-to-a-previous-kernel-version/72820/7
https://fedoramagazine.org/install-kernel-koji

Thanks

Reply:
Koji is not available of the Centos Stream 8 repo:
```sh
[root@ibm-p8-kvm-03-guest-02 ~]# dnf install koji
...
No match for argument: koji
Error: Unable to find a match: koji
```

Reply:
What I just found out is that there's a kojihub for Centos Stream 8 and it has a build with the new kernel [here](https://kojihub.stream.centos.org/koji/buildinfo?buildID=33049)

FYI @brianmcarey

Reply:
I have created a PR using the box from CentOS kojihub here: https://github.com/kubevirt/kubevirtci/pull/1012